### ISSUE TITLE
Support for new Contacts features: photo, groups, extended properties, batch

### ISSUE THREAD

**Issue by GoogleCodeExporter** [barsu8184/google-gdata#129]
```
See release anouncement of new features:
http://googledataapis.blogspot.com/2008/05/google-contacts-api-update.html
```

Original issue reported on code.google.com by `api.jscu...@gmail.com` on 2 Jun 2008 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
I need the photo feature as well.
```

Original comment by `thomasar...@gmail.com` on 5 Jun 2008 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed with version 1.2.0.0 
```

Original comment by `fman...@gmail.com` on 10 Jun 2008 at 4:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Great. The photo uri is no longer null. Could you update the following page 
with a
code sample?:

http://code.google.com/apis/contacts/developers_guide_dotnet.html
```

Original comment by `thomasar...@gmail.com` on 10 Jun 2008 at 4:35
